rust/src/dcerpc/dcerpc.rs

```diff
@@ -301,6 +300,10 @@ pub struct DCERPCState {
     pub prev_dir: u8,
     pub prev_tx_call_id: u32,
     pub clear_bind_cache: bool,
+    pub skip_ts: u32,
```

Comment:
unused?

Reply:
Yes. Had introduced it earlier trying to track the skip data, forgot removing.